native-platform/src/main/cpp/win.cpp

```diff
@@ -364,7 +363,7 @@ Java_net_rubygrapefruit_platform_internal_jni_PosixFileSystemFunctions_listFileS
                 if (GetVolumeInformationW(cur, NULL, 0, NULL, NULL, &flags, fileSystemName, MAX_PATH + 1) == 0) {
                     env->CallVoidMethod(info, unknownFsMethod,
                         mount_point,
-                        file_system_type,
+                        NULL,
```

Comment:
I think I'd use something like `unknown` here instead to avoid having to deal with `null`. WDYT?

Reply:
I am using `unknown` now, though I deal with the problem on the Java side and continue to return `null` here, so I don't have to deal with strings in C.